meeting-notes/2023-12-09.md

```diff
@@ -0,0 +1,43 @@
+# Encryption
+
+### Preface
+
+Before we start, we assume Alice wants to communicate a message with Bob.
+We also assume that Eve wants to know what they are communicating and hears everything Alice and Bob tell each other.
+Alice and Bob would prefer that Eve does not hear what they are communicating. (maybe they are planning a surprise party)
+
+
+### The method we are learning about
+
+1) Pick $g$ (which is called the generator) and $p$ (which is called the prime number)
+	- Both large primes
+	- Alice, Bob, and Eve see this
+2) Alice picks $P_{rivateA}$
+	- must be at least 5 digits
+	- only Alice knows this
+3) Bob picks $P_{rivateB}$
+	- must be at least 5 digits
+	- only Bob knows this
+4) Alice calculates $P_{ublicA} = g^{P_{rivateA}} \mod {p}$

```

Comment:
Usually, $P_{uplicA}$ is referred to as $A$ and ${P_{rivateA}}$ is referred to as $a$. Likewise $P_{uplicB}$ is usually $B$ and ${P_{rivateB}}$ is usually $b$.